packages/react-components/react-storybook-addon-export-to-sandbox/package.json

```diff
@@ -2,7 +2,6 @@
   "name": "@fluentui/react-storybook-addon-export-to-sandbox",
   "version": "0.1.0",
   "description": "Storybook addon that enables cloud/web sandbox exports for stories doc mode code examples",
-  "private": true,
```

Comment:
we need to switch to caret deps to avoid dupes 

<img width="605" alt="image" src="https://github.com/user-attachments/assets/0746ef35-87a6-47a6-ae77-aba9408e0247" />